sdks/python/apache_beam/runners/direct/clock.py

```diff
@@ -41,7 +41,7 @@ def time(self):
 
 class TestClock(object):
   """Clock used for Testing"""
-  def __init__(self, current_time=0):
+  def __init__(self, current_time=time.time()):
```

Comment:
<!--new_thread; commit:a769afcabd2a4b8ad94cae1f1c62e73823e42d7b; resolved:0-->
Let's just use 0 so the tests execute more deterministically.  Note that with your current code, the default is evaluated once (i.e. when this file is loaded), not on every call to `__init__` here, as you may have intended.